client/app/app.js

```diff
@@ -0,0 +1,51 @@
+import { h, Component } from 'preact'
+import { Router } from 'preact-router'
+import { I18n } from './plugins/preact-polyglot'
+
+import Sidebar from './components/sidebar'
+import Discovery from './components/discovery'
+import CategoryList from './components/category_list'
+import ConnectedList from './components/connected_list'
+import ItemDialog from './components/item_dialog'
+
+export default class App extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            konnectors: props.konnectors,
+            selectedItem: null
+        }
+        this.onItemSelect = this.onItemSelect.bind(this)
```

Comment:
We still use this classic binding way or we plan to use a decorator for this? This way is good for me, it's just to keep coherent. Or maybe we can use the decorator later if necessary.